products/views.py

```diff
@@ -1,3 +1,61 @@
-from django.shortcuts import render
+import re
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Count
+
+from .models          import (
+    Category,
+    CategoryProduct,
+    Product,
+    ProductGroup,
+    ProductGroupImage,
+    ProductGroupPackageType,
+    Relation
+)
+
+class CategoryView(View):
+    def get(self, request):
+        categories = Category.objects.all()
+
+        category = [{
+            "id" : category.id,
+            "name" : category.name
+        } for category in categories]
+
+        return JsonResponse({"category": category}, status=200)
+
+class ProductListView(View):
```

Comment:
반환되는 resource가 Product가 아니라 ProductGroup이기 때문에 아래와 같이 수정이 필요해보입니다.
```suggestion
class ProductGroupListView(View):
```